Storage/rest/hypermedia.h

```diff
@@ -55,32 +55,31 @@ CURRENT_STRUCT(HypermediaRESTStatus) {
 
 CURRENT_STRUCT(HypermediaRESTError) {
   using T_DETAILS = std::map<std::string, std::string>;
-  CURRENT_FIELD(error, std::string);
+  CURRENT_FIELD(name, std::string);
   CURRENT_FIELD(description, std::string);
```

Comment:
This `description` is often called a `message` -- even the generic response has a `message`, not `description`. Consistency?